### PR TITLE
Laboratory work #3, Anna Belyavskaya - 23FPL2

### PR DESCRIPTION
**Pull request by AnnaBeliavskaya**

### CODE REVIEW

**Review by EIzharikov — COMMENTED** [review]
Nice start, almost nothing to review, you should hurry up! This lab is hard! Make CI green and continue your work!

**Review by EIzharikov — COMMENTED** [review]
only for 4? You can reach 6 at least!

File: lab_3_ann_retriever/start.py [hunk 1]
```diff
@@ -30,7 +32,10 @@ def main() -> None:
     """
     with open("assets/secrets/secret_1.txt", "r", encoding="utf-8") as text_file:
         text = text_file.read()
-    result = None
+    documents, stop_words = open_files()
+    tokenizer = Tokenizer(stop_words)
+    preprocessed_documents = tokenizer.tokenize_documents(documents)
+    result = preprocessed_documents
```
Comment (EIzharikov) [inline lab_3_ann_retriever/start.py:38]: show here more your implementation

**Review by EIzharikov — COMMENTED** [review]
Nice! You reached 6! Be ready for Friday!

File: lab_3_ann_retriever/start.py (hunk 1 above)
Comment (AnnaBeliavskaya) [reply lab_3_ann_retriever/start.py]: done!

**Pull request closed without merging**